src/handlers/itemStore.ts

```diff
@@ -121,5 +121,90 @@ export const itemStoreHandlers: NatsHandler[] = [
                 }
             }
         }
-    ]
+    ],
+    [
+        "item_store_inventory_add_item",
```

Comment:
this is good. We should now have a `src/private/inventory.ts` file with inventory related specific handlers (add, update inventory item), just like we have one for warehouse items here: https://github.com/jwa-lab/item-store/pull/8/files#diff-47395904df7cd6f24caacb52d40919122dea3fa8b148297fa35ed196a737a0d4R2

Reply:
Also, the subject name should be something like `add_inventory_item`. No need to prefix with `item_store`, this is now done by the registerHandler util: https://github.com/jwa-lab/item-store/pull/8/files#diff-693123dee3ea7170042205e2bcf2cdf9098be01b36872438a25d65d99ad090c5R33

And the resulting subject name will be `item_store.add_inventory_item`, you'll have to use this syntax in your tests